packages/formation-react/src/components/Telephone/Telephone.stories.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {
+  Title,
+  Subtitle,
+  Description,
+  Primary,
+  ArgsTable,
+  Stories,
+} from '@storybook/addon-docs/blocks';
+
+import Telephone, { CONTACTS } from './Telephone';
+import { contactsMap } from './contacts';
+import Table from '../Table/Table';
+
+const fields = [
+  { label: 'Property name (CONTACTS.x)', value: 'key' },
+  { label: 'Phone number', value: 'value' },
+  { label: 'Description', value: 'description' },
+];
+const Contacts = () => (
+  <Table
+    fields={fields}
+    data={Object.entries(contactsMap).map(c => ({
+      key: c[0],
+      value: <Telephone contact={c[1].phoneNumber} />,
+      description: <div style={{ maxWidth: '30em' }}>{c[1].description}</div>,
+    }))}
+  />
```

Comment:
A bit of space after this would help things flow better:

**Before:**

![image](https://user-images.githubusercontent.com/2008881/98976986-e4791180-24cc-11eb-8551-fb50c7690579.png)


**After:**

![image](https://user-images.githubusercontent.com/2008881/98977027-f5c21e00-24cc-11eb-9cf7-9b6b85fb0df0.png)


```suggestion
  <>
    <Table
      fields={fields}
      data={Object.entries(contactsMap).map(c => ({
        key: c[0],
        value: <Telephone contact={c[1].phoneNumber} />,
        description: <div style={{ maxWidth: '30em' }}>{c[1].description}</div>,
      }))}
    />
    <br />
  </>
```